add navbar and font

Leeoku opened the pull request:
Need help aligning the navbar text to the "learnFE" text

estebanborai commented on index.html:
```diff
@@ -1,20 +1,30 @@
+
 <body>
   <header>
+
```
Avoid blank lines please

estebanborai commented on node_modules/normalize.css/normalize.css:
```diff
@@ -0,0 +1,341 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```
Normalize.css is alreay set, you can find it here: `website/styles/normalize.css` theres no need to use **npm package manager***.

estebanborai commented on style/header.css:
```diff
@@ -28,6 +28,21 @@ h1 {
+  font-size: 1.5rem;
+
+}
+a:visited{
```
This line will turn every visited link into #f0f0f0, please define this rule so the white links are in the right section.

estebanborai commented on node_modules/normalize.css/package.json:
```diff
@@ -0,0 +1,66 @@
+{
```
To install **npm** packages I recommend you to install Node and follow the instructions to init a npm module and how to install a package. Anyway we are not using NPM or any other dependencies such as jQuery, Bootstrap, etc in this project.

Leeoku commented:
Fixes:
Removed the npm files/packages/unnecessary files
Removed empty lines
Added rules to anchor colour in header.css

Issues
Need help aligning text from navbar to learnFE logo text
After putting navbar, the position of navbar text varies due to the script of changing words. I put a countermeasure of width:5vw in  #logocontainer, but not sure if this is best way. Please confirm!

estebanborai commented:
Using `vw` its not a good idea for sizing such elements.
Consider using fixed width instead. 
Try to declare a size that covers the largest text.
